types/lib/model.d.ts

```diff
@@ -2077,6 +2077,33 @@ export abstract class Model<T = any, T2 = any> extends Hooks {
     options: IncrementDecrementOptions
   ): Promise<M>;
 
+  /**
+   * Decrements a single field.
+   */
+  public static decrement<M extends Model, K extends keyof M>(
+    this: { new(): M },
+    field: K,
+    options: IncrementDecrementOptionsWithBy
+  ): Promise<M>;
```

Comment:
I actually believe the docs (and typings on increment) are wrong.

The static return signature appears to be
```return promise.then(affectedRows => {
      if (options.returning) {
        return [affectedRows, affectedRows.length];
      }

      return [affectedRows];
    });
```

so `[number] | [unknown[], number]`

Reply:
@SimonSchick I haven't looked into the actual `static increment` and `static decrement` source code yet. I'll look into it later this week then.

Reply:
@SimonSchick I have re-committed an update to my PR. Feel free to review it.